nix/default.nix

```diff
@@ -229,13 +229,56 @@ let
           };
           propagatedBuildInputs = [ fmt astring ];
         }) { };
+    pprint = self.callPackage
+      ({ stdenv, fetchFromGitHub, buildDunePackage, ocamlbuild }:
+        buildDunePackage rec {
+          pname = "pprint";
+          version = "20180528";
+
+          minimumOCamlVersion = "4.03";
+          src = fetchFromGitHub {
+            owner = "fpottier";
+            repo = "${pname}";
+            rev = "${version}";
+            sha256 = "1jhmmd7ik1lx9y5niqv5rknhq02pkwmyxc5c0wndp5cyp8hsj0py";
+          };
+          buildInputs = [ ocamlbuild ];
+          buildPhase = "";
+          installPhase = ''
+            mkdir -p $out/lib/ocaml/4.07.1/site-lib
+            make install
+          '';
+          doCheck = false;
+        }) { };
+    ocp-ocamlres = self.callPackage
+      ({ stdenv, fetchFromGitHub, buildDunePackage, pprint, astring, base }:
+        buildDunePackage rec {
+          pname = "ocp-ocamlres";
+          version = "0.4";
+
+          minimumOCamlVersion = "4.03";
+          src = fetchFromGitHub {
+            owner = "OCamlPro";
+            repo = "${pname}";
+            rev = "v${version}";
+            sha256 = "0smfwrj8qhzknhzawygxi0vgl2af4vyi652fkma59rzjpvscqrnn";
+          };
+          buildInputs = [ pprint astring base ];
+          buildPhase = "";
+          installPhase = ''
+            mkdir -p $out/lib/ocaml/4.07.1/site-lib
+            mkdir -p $out/lib/ocaml/4.07.1/site-lib//../bin
```

Comment:
?!?!

Reply:
probably `//` should be `/`, but `ocp-ocamlres` really wants to have `/../bin` in order to install binary
https://github.com/OCamlPro/ocp-ocamlres/blob/master/Makefile#L2